contact.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="assets/css/style.css">
+  <title>Contact</title>
+</head>
+
+<body>
+
+  <nav>
+    <div class="name">Tanya Schwartz</div>
+    <div class="navPills">
+      <div>
+        <a href="About">About</a>
```

Comment:
your about page is giving an error when you click on it from the contact and portfolio page because the file path is incorrect